### ISSUE TITLE
Plugin Tests scaffolding and installing on Windows

### ISSUE THREAD

**Issue by ethanclevenger91** [wp-cli/wp-cli#2359]
Has anyone tried getting this to work on Windows? I'm getting the following:

```
bash bin/install-wp-tests.sh wordpress_test root password localhost latest
+ install_wp
+ '[' -d /tmp/wordpress/ ']'
+ return
+ install_test_suite
++ uname -s
+ [[ MINGW64_NT-10.0 == \D\a\r\w\i\n ]]
+ local ioption=-i
+ '[' '!' -d /tmp/wordpress-tests-lib ']'
+ cd /tmp/wordpress-tests-lib
+ '[' '!' -f wp-tests-config.php ']'
+ install_db
+ PARTS=(${DB_HOST//\:/ })
+ local PARTS
+ local DB_HOSTNAME=localhost
+ local DB_SOCK_OR_PORT=
+ local EXTRA=
+ '[' -z localhost ']'
++ echo
++ grep -e '^[0-9]\{1,\}$'
+ '[' ']'
+ '[' -z ']'
+ '[' -z localhost ']'
+ EXTRA=' --host=localhost --protocol=tcp'
+ mysqladmin create wordpress_test --user=root --password=password --host=localhost --protocol=tcp
```

And then it doesn't install the `/tmp/` directory.

**Comment by danielbachhuber** [comment]
WP-CLI doesn't fully support Windows — it's a bit "Your Mileage May Vary".

We'd be able to better determine the proper temp directory with #2119, as a part of the improvements that landed in #2353

**Comment by ethanclevenger91** [comment]
I'll keep an eye out then. Thanks!
